.gitignore

```diff
@@ -9,7 +9,7 @@ node_modules
 telemetryCache
 quickStart.html
 .gitcommit
-media/libs/lodash.min.js
+media/libs/
```

Comment:
There's a file checked in at this location. Is this change a necessary part of this PR?

Reply:
No, this is not a necessary change.